### PR TITLE
fix parsing of owner and repo

### PR DESCRIPTION
**Pull request by sinisastanic**
closes [#655](https://github.com/OctoMind-dev/automagically/issues/665)

### CODE REVIEW

**Comment by sinisastanic** [comment]
@RunOrVeith solving the regex problem 🥰

File: Jenkinsfile [hunk 1]
```diff
@@ -11,10 +11,12 @@ pipeline {
...
-                    String[] split_url = "${env.GIT_URL}".split('/')
-                    int number_of_elements = split_url.size()
-                    final String repo = split_url[number_of_elements - 1]
-                    final String owner = split_url[number_of_elements - 2]
+                    
+                    def matchesOrgAndRepo =  ("https://github.com/OctoMind-dev/jenkins-integration.git" =~ ".*(/|:)(.*)/([^\\.]*)(\\.git)?\$")
+                    final String owner = matchesOrgAndRepo[0][2]
+                    final String repo = matchesOrgAndRepo[0][3]
+
+                    matchesOrgAndRepo = null
```
Comment (Germandrummer92) [inline Jenkinsfile:19]: why this assignment? 🤔

**Review by Germandrummer92 — CHANGES_REQUESTED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -11,10 +11,12 @@ pipeline {
                     final String baseUrl = "https://app.octomind.dev"
                     final String url = "${baseUrl}/api/v2/execute"
                     final String header = "Content-Type: application/json"
-                    String[] split_url = "${env.GIT_URL}".split('/')
-                    int number_of_elements = split_url.size()
-                    final String repo = split_url[number_of_elements - 1]
-                    final String owner = split_url[number_of_elements - 2]
+                    
+                    def matchesOrgAndRepo =  ("https://github.com/OctoMind-dev/jenkins-integration.git" =~ ".*(/|:)(.*)/([^\\.]*)(\\.git)?\$")
```
[the remark this answers is not in the document]
Comment (Germandrummer92) [reply Jenkinsfile:15]: also: maybe we just send the full git url to the backend and do the splitting/url building there? seems a lot more generic and we don't need to change the jenkinsfile every time?

**Review by sinisastanic — COMMENTED** [review]
successful run [here](http://35.246.255.232:8080/job/GitHub%20Example%20Pipeline/62/console)

**Review by Germandrummer92 — APPROVED** [review]

**Pull request merged**